ddcommon-ffi/src/slice.rs

```diff
@@ -53,6 +54,19 @@ pub type CharSlice<'a> = Slice<'a, c_char>;
 /// Use to represent bytes -- does not need to be valid UTF-8.
 pub type ByteSlice<'a> = Slice<'a, u8>;
 
+// Send, Sync, UnderlyingBytes, and AsRef<[u8]> are all implemented to support using ByteSlice as
```

Comment:
What's the difference between, using a ByteSlice<'static> as underlying and calling `Bytes::from_static` on the `&'static [u8]`

Reply:
We can't call `Bytes:from_static` in `ddog_trace_exporter_send` because we want `trace_exporter::send` to accept a type that implements `UnderlyingBytes`, not the Bytes themselves.

I guess we could implement `UnderlyingBytes` for `&'static [u8]`. I slightly prefer implementing it, and the associated traits for `ByteSlice` as it's a bit more intentional, but I don't feel strongly about it.

Reply:
Removed this and went with using `Bytes::from_static`